docs/git.rst

```diff
@@ -0,0 +1,304 @@
+.. -*- coding: utf-8 -*-
+
+==================
+git best practices
+==================
+Collection of best practices that would help you and the overall community to work with git more efficiently.
+
+General guidelines
+==================
+
+Pulling code
+------------
+*ALWAYS* (in capital letters) do a :command:`git pull -r` when fetching new code,
```

Comment:
I agree with this point, but as @do3cc says, you should possibly mention what the argument does. Even just mentioning the word 'rebase' might be enough.